webapp/security/views.py

```diff
@@ -31,8 +31,22 @@
 session = talisker.requests.get_session()
 
 
+def get_processed_details(notice):
+    pattern = re.compile(r"(cve|CVE-)\d{4}-\d{4,7}", re.MULTILINE)
+
+    return re.sub(
+        pattern, r'<a href="/security/\g<0>">\g<0></a>', notice["description"]
+    )
+
+
 def notice(notice_id):
-    notice = db_session.query(Notice).get(notice_id)
+
+    security_api = SecurityAPI(
```

Comment:
Could you please move the `security_api` from the `cve` function to the global part of this file, so it can be used by both views?